app/views/case_contacts/form/details.html.erb

```diff
@@ -78,18 +78,28 @@
     </div>
 
     <% if @case_contact.contact_topic_answers.any? %>
-    <div class="card-style-1 pl-25 mb-10">
-      <h4 class="mb-20 details__topics-label">Court report <%= "topic".pluralize(@case_contact.contact_topic_answers.count) %> <span class="content-1">(optional)</span></label></h4>
-      <div class="">
-        <%= form.fields_for(:contact_topic_answers) do |f| %>
-          <div class="form-check checkbox-style mb-10 ml-5">
-            <%= f.check_box :selected, class: ["form-check-input", "casa-case-id"] %>
-            <%= f.label :selected, f.object.contact_topic.question, class: "form-check-label" %>
-          </div>
+      <div class="card-style-1 pl-25 pr-25 mb-10">
+        <h4 class="mb-20 details__topics-label">Court report <%= "topic".pluralize(@case_contact.contact_topic_answers.count) %> <span class="content-1">(optional)</span></h4>
+        <div>
+          <%= form.fields_for(:contact_topic_answers) do |f| %>
+            <div class="form-check checkbox-style mb-10 ml-5">
+              <%= f.check_box :selected, class: ["form-check-input", "casa-case-id"] %>
+              <%= f.label :selected, f.object.contact_topic.question, class: "form-check-label" %>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    <% elsif @case_contact.contact_topic_answers.none? %>
+      <div class="card-style-1 pl-25 pr-25 mb-10">
+        <h4 class="mb-20 details__topics-label">Court Report Topics</h4>
+        <% if current_user.volunteer? || current_user.supervisor? %>
+          <div class="mb-10 ml-5">Your organization has not set any Court Report Topics yet. Contact your admin to learn more.</div>
+        <% elsif current_user.casa_admin? %>
+          <div>Visit <%= link_to "Manage Case Contact Topics", edit_casa_org_path(current_user.casa_org_id) %> to set your organization Court report topics.</div>
```

Comment:
This seems like a lot of duplication of HTML. I think we can make the if statement only dictate this part:

```erb
  <%= form.fields_for(:contact_topic_answers) do |f| %>
    <div class="form-check checkbox-style mb-10 ml-5">
      <%= f.check_box :selected, class: ["form-check-input", "casa-case-id"] %>
      <%= f.label :selected, f.object.contact_topic.question, class: "form-check-label" %>
    </div>
  <% end %>
```

Basically
- if there are topics show that part
- if there are none use the admin/volunteer logic you have.
- lets make the volunteer text be the default. you can make that the else clause instead of using if / elsif

I also think I like this as the heading more than what I had before:

```erb
 <h4 class="mb-20 details__topics-label">Court Report Topics <span class="content-1">(optional)</span></h4>
```

Reply:
Adjusted! Thank you for the suggestions - let me know if you see something that doesn't look right.